docs/src/pages/index.tsx

```diff
@@ -183,33 +266,40 @@ function Features() {
   return (
     <div className="container margin-top--lg margin-bottom--md">
       <Heading as="h1" className="margin-bottom--md text--center text--primary">
-        Features
+        Capabilities
       </Heading>
       <p className={clsx('text--center', styles.sectionSubheading)}>
         Simple, powerful, fast. Pick three.
       </p>
 
       <Feature
-        text="WebGPU APIs"
-        secondaryText="Utilize existing WebGPU-based components or craft your own WGSL shader to achieve any desired effect and directly leverage GPU parallel processing capabilities."
+        text="Built-in components"
+        secondaryText="Use built-in components to process streams, render text, images, GIFs or websites, and combine them into output streams."
+        image={<FaPuzzlePiece className={styles.featureIcon} style={{ fontSize: 80 }} />}
+      />
+
+      <Feature
+        text="Highly customizable"
+        secondaryText="Create custom effects with WGSL shaders and directly leverage GPU parallel processing capabilities."
         image={wgpuLogo}
       />
+
+      <Feature
+        text="Smooth transitions"
+        secondaryText="Smoothly transition layouts with animations using built-in transition mechanism."
+        image={<GiFeather className={styles.featureIcon} style={{ fontSize: 80 }} />}
+      />
+
       <Feature
         text="Real-time processing"
         secondaryText="Process live video streams in real-time with low latency. Blazingly fast processing makes video conferencing, live-streaming, and broadcasting an everyday bread and butter for Live Compositor."
```

Comment:
I would consider removing "Blazingly fast processing makes video conferencing, live-streaming, and broadcasting an everyday bread and butter for Live Compositor.". It sounds a bit weird.